src/screens/DetailScreen.js

```diff
@@ -83,7 +83,8 @@ const useRootRouteByCategory = (details, navigation) => {
 export const DetailScreen = ({ navigation, route }) => {
   const { isConnected, isMainserverUp } = useContext(NetworkContext);
   const query = route.params?.query ?? '';
-  const queryVariables = route.params?.queryVariables ?? {};
+  const detailId = route.params?.id ?? undefined;
```

Comment:
is `id` already in use here, so you have chosen `deatilId`?
there is no `?? undefined;` needed. if there is no `route.params?.id` it is undefined.

Reply:
I thought `detailId` was a better nomenclature, but since we are already in `DetailScreen`, id nomenclature can also be used.
Yes, that's right. I will update this. 👍